plugins/woocommerce/i18n/states.php

```diff
@@ -1186,6 +1186,21 @@
 		'OT' => __( 'Otago', 'woocommerce' ),
 		'SL' => __( 'Southland', 'woocommerce' ),
 	),
+	'PA' => array( // Panama states.
+		'PA-1' => __( 'Bocas del Toro', 'woocommerce' ),
+		'PA-2' => __( 'Coclé', 'woocommerce' ),
+		'PA-3' => __( 'Colón', 'woocommerce' ),
+		'PA-4' => __( 'Chiriquí', 'woocommerce' ),
+		'PA-5' => __( 'Darién', 'woocommerce' ),
+		'PA-6' => __( 'Herrera', 'woocommerce' ),
+		'PA-7' => __( 'Los Santos', 'woocommerce' ),
+		'PA-8' => __( 'Panamá', 'woocommerce' ),
+		'PA-9' => __( 'Veraguas', 'woocommerce' ),
+		'PA-10' => __( 'Panamá Oeste', 'woocommerce' ),
```

Comment:
I believe this should be the English name, "West Panamá".

Reply:
Thanks @jeffstieler for all the comments and checks for the PRs. Appreciate the feedback.